Reset PW / Email Notification

leong96 opened the pull request:
This PR implements two features:

- When an email address is inputted into resetPW, no matter what was submitted in there, a message will pop up indicating that if the email is associated with a provider account, an email will be sent.

- When a user is approved, the email address as well as the alt. contact will get emails. Going to stall on the rejection email until we figure out what we're doing about rejection messages.

leong96 committed:
Reset PW  / Email Notification

This PR implements two features:

leong96 committed:
Revert "Reset PW  / Email Notification"

This reverts commit 0cfe792eaa287b742fff95086244eb1830d37356.

leong96 committed:
Revert "Revert "Reset PW  / Email Notification""

This reverts commit 354917ec773aecda1a639010aeff5ee4180e6dc2.

leong96 committed:
Update resetPW.html

Has -> Will, ah, grammar~

R-Eli-Levasseur commented on grasa_event_locator/views.py:
```diff
@@ -376,15 +376,16 @@ def resetpw(request):
                         print(send_email([request.POST['emailAddr']], "GRASA - Reset Password", "You've requested a password reset at the GRASA Event Locator. Please visit this linnk: http://grasa.larrimore.de/resetPWForm/" + resetlink))
-
-        return render(request, 'resetPW.html')
+        context = {'email_sent': True}
```
You set email_sent to true in both places, so I believe the message is always gonna show up

leong96 committed:
email_sent -> email_submitted

Rewrote logic so that the context that manages the reset password email notification only applies after a post submit, not at all times.

Also renamed email_sent to email_submitted to avoid confusion.
